erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md

```diff
@@ -0,0 +1,52 @@
+<!-- add-breadcrumbs -->
+
+**Invoice discounting is the practice of using a company's unpaid invoices as collateral for a short term loan, which is issued by a bank or a finance company.**
+
```

Comment:
add a paragraph demonstrating a scenario to add clarity on use case
like: consider that you have some unpaid invoices for xyz and you need money...etc

Reply:
I have covered that in the definition...